R/get_warnings.R

```diff
@@ -0,0 +1,33 @@
+#' Get the warnings of qenv object
+#'
+#' @param object (`qenv`)
+#'
+#' @return character vector of warnings
+#' @export
+#'
+#' @examples
+#' data_q <- new_qenv() %>% eval_code("iris_data <- iris")
+#' warning_qenv <- eval_code(
+#'   data_q,
+#'   bquote(p <- hist(iris_data[, .("Sepal.Length")], ff = ""))
+#' )
+#' get_warnings(warning_qenv)
+#'
+get_warnings <- function(object) {
+  attr(object@code, "warning")
```

Comment:
we should also make sure joining qenv's together transfers the warning and messages as well

Reply:
done